src/js/components/Base.js

```diff
@@ -7,8 +7,13 @@ import Link from './Link'
 import MaxWidth from './MaxWidth'
 
 const Header = ({ currentUrl }) => {
-  const linkClasses = 'no-underline -mx-1 p-1 rounded hover:bg-gray-100'
-  const links = [{ href: '/info/', title: 'Info' }]
+  const linkClasses =
+    'no-underline -mx-1 p-1 rounded hover:bg-gray-100 whitespace-nowrap'
+  const links = [
+    { href: '/info/', title: 'Info' },
+    { href: 'https://twitter.com/koodikrapula', title: 'Twitter' },
+    { href: 'https://forms.gle/uyh3v8bbQUwnizEw5', title: 'Kysy kysymys' },
+  ]
```

Comment:
Idea: use the "external-link" icon from [Heroicons](https://heroicons.com/) after the external links' texts. Wdyt?

(Asking this question again because non-threaded questions are easy to miss.)